predicators/utils.py

```diff
@@ -3532,6 +3532,28 @@ def f_range_intersection(lb1: float, ub1: float, lb2: float,
     return (lb1 <= lb2 <= ub1) or (lb2 <= lb1 <= ub2)
 
 
+def compute_abs_bounds_given_frange(lb1: float, ub1: float, lb2: float,
```

Comment:
it seems like this function doesn't necessarily involve "features", so can we change the name to remove the 'f'? let's also do so for `f_range_intersection` above, while we're at it